lib/tarantool_server.py

```diff
@@ -42,6 +37,13 @@
 from lib.utils import prefix_each_line
 from test import TestRunGreenlet, TestExecutionError
 
+try:
+    # Python 2
+    from StringIO import StringIO
+except ImportError:
+    # Python 3
+    from io import StringIO
```

Comment:
I would keep 'platform' imports first and 'site modules' imports next. This is generally followed in the test-run code, but not everywhere. Why do you moved this import downside? Why only this one?

I had guess that it is to work around 'E402 module level import not at top of file' flake8 warning, but it seems it does not complain against imports on this kind.

Reply:
Moved StringIO import before 'site modules' imports.